src/widgets/tabs/tabs.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from 'react';
+import styles from './tabs.module.css';
+
+export interface Tab {
+  label: string;
+  content: React.ReactNode;
+}
+
+interface TabsProps {
+  tabs: Tab[];
+}
+
+export function Tabs({ tabs }: TabsProps) {
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  return (
+    <div className={styles.tabs}>
+      <div className={styles.tabsHeader}>
+        {tabs.map((tab, index) => (
+          <button
+            key={index}
+            className={`${styles.tabButton} ${index === activeIndex ? styles.active : ''}`}
```

Comment:
```suggestion
            className={{index === activeIndex ? `${styles.tabButton} ${styles.active}` : styles.tabButton}
```
This solution fixes "class-name " < blank space at the end of class name.
No important.

Reply:
Good suggestions! I think all of these were done